### ISSUE TITLE
MinifyJavaScripts rejects scripts as not local if served via javascript_include_tag

### ISSUE THREAD

**Issue by plindberg** [juliocesar/rack-pagespeed#13]
Due to the timestamp tacked to the path to the script, the regexp in MinifyJavaScripts doesn't match. I solved it by creating an own filter as a subclass, overriding `local_script` as follows:

```
def local_script? node
  node['src'] and !(node['src'] =~ /^http/ or !(node['src'] =~ /\.js(\?\d+)?$/))
end
```

**Comment by juliocesar** [comment]
The correct way to solve the problem is to have minify_javascripts do what combine_javascripts does, which is

```
def local_script? node
  node.name == 'script' and file_for(node)
end
```

The point of that method is to establish whether the script is local or not, and the incorrect way I did was to write a regex to find that out, rather than checking if the file can be found. I fixed that problem with combine_javascripts and forgot to update minify_javascripts.

I'll release 1.0.5 tomorrow probably with that fix.
